ops/roles/infra/tasks/main.yml

```diff
@@ -55,5 +55,7 @@
   # See https://github.com/puppeteer/puppeteer/blob/main/docs/troubleshooting.md#recommended-enable-user-namespace-cloning
 - name: Enable user namespace cloning to allow running Chromium in a sandbox
   command: sysctl -w kernel.unprivileged_userns_clone=1
+  tags: [skipAppleSilicon]
```

Comment:
I suggest to do something transparent for the user, and avoid the need to pass the option `skipAppleSilicon` as it can be programmatically found:
`tags: [skipAppleSilicon]` ==> `when: ansible_facts['architecture'] != "aarch64"`